otx/algorithms/anomaly/configs/classification/draem/template_experimental.yaml

```diff
@@ -1,5 +1,5 @@
 # Description.
-model_template_id: ote_anomaly_classification_draem
+model_template_id: otx_anomaly_classification_draem
```

Comment:
Please don't change model_template_id as it will cause many errors in Geti side.